xdsl/dialects/transform.py

```diff
@@ -622,16 +626,16 @@ def __init__(
         self,
         target: SSAValue,
         dynamic_sizes: Sequence[SSAValue],
-        static_sizes: DenseArrayBase | Sequence[int] | None = None,
-        interchange: DenseArrayBase | Sequence[int] | None = None,
-        scalable_sizes: (DenseArrayBase | Sequence[int] | None) = None,
+        static_sizes: DenseArrayBase[I64] | Sequence[int] | None = None,
```

Comment:
I'm not sure the `Annotated` type does anything here

Reply:
Yeah it doesn't I'll remove it